apps/mark-scan/frontend/src/pages/diagnostics/diagnostics_screen.tsx

```diff
@@ -108,6 +131,17 @@ export function DiagnosticsScreen({
           </Main>
         </Screen>
       </Route>
+      <Route path="/paper-handler">
+        <PaperHandlerDiagnosticScreen
+          onClose={() => {
+            history.push('/');
+            // The diagnostic record is written by the backend after successful rear ejection.
+            // Invalidating the query at the time of the last mutation in this flow is still too early
+            // so we have to manually refetch.
+            void mostRecentPaperHandlerDiagnosticQuery.refetch();
```

Comment:
Any reason to refetch here rather than invalidate?

Reply:
In testing, invalidation resulted in stale data being rendered. Only a hard refetch fixed it. I reviewed the state machine and I think this comment is outdated because the diagnostic record is written on entry to `success` or `failure` states.

I can play with this some more when I'm in front of the hardware next week but would prefer to leave this as-is since we are guaranteed it'll work.

Reply:
Gotcha. I would recommend at least using `await` instead of `void`, since `void` swallows errors